lua/wire/client/cl_wirelib.lua

```diff
@@ -162,16 +162,19 @@ function Wire_DrawTracerBeam( ent, beam_num, hilight, beam_length )
 	if (beam_length > 0) then
 
 		local x, y = 0, 0
-		if (ent.GetSkewX and ent.GetSkewY) then
+		if ent.GetTarget then
+			if ent:GetTarget().X ~= 0 or ent:GetTarget().Y ~= 0 or ent:GetTarget().Z ~= 0 then
+				ent.endpos = ent:GetPos() + ( ent:GetTarget() - ent:GetPos() ):GetNormalized()*beam_length
```

Comment:
Need the `ent.endpos[1] ~= ent.endpos[1]` check here too

Reply:
Why would you check if something is different from itself?

Reply:
We also have https://github.com/wiremod/wire/blob/master/lua/wire/wireshared.lua#L949-L951 if you'd like to use that